index.js

```diff
@@ -0,0 +1,77 @@
+'use babel';
+
+import path from 'path';
+import { execNode, find } from 'atom-linter';
+
+const GRAMMAR_SCOPES = [
+  'text.html.angular',
+  'text.html.basic',
+  'text.html.erb',
+  'text.html.gohtml',
+  'text.html.jsp',
+  'text.html.mustache',
+  'text.html.handlebars',
+  'text.html.ruby'
+];
+
+export const config = {
+  executablePath: {
+    title: 'Executable Path',
+    description: 'HTMLHint Executable Path',
+    type: 'string',
+    default: path.join(__dirname, '..', 'node_modules', 'htmlhint', 'bin', 'htmlhint')
+  }
+};
+
+const executablePath = () => atom.config.get('linter-htmlhint.executablePath');
```

Comment:
`atom.config.observe()` this setting in activate instead of always `get()`ing it.

If you've never stepped through `get()` there is quite a bit of work that goes into parsing the options, validating them, and figuring out what to return before you get a value back. If you `observe()` the setting it does the work _once_, and then will automatically update the value for you when, if ever, the user changes it.